src/app.ts

```diff
@@ -15,6 +15,8 @@ if (!server) {
   server = getServerDomain().replace(app, '');
 }
 
+const mock = parameters.has('mock');
+const miningUrl = mock ? 'http://localhost:3000/mock.json' : 'tbd';
```

Comment:
maybe we should rename this parameter already to something like `miningDataUrl`.
And change the behaviour to: if you give a parameter we use this URL, if not we use the internal mock data url.